pydoc.el

```diff
@@ -68,6 +68,12 @@
   :group 'pydoc)
 
 
+(defcustom pydoc-pip-command "pip --version"
```

Comment:
This doesn't seem like the right approach. It would be better to provide an options variable. Here you combine command and options. If pydoc-pip-command is only used for getting the version, then I think probably the variable name should have version in it, so that later someone doesn't think pydoc-pip-command means the command for using pip.